CLDR-18410 ddl landing page

srl295 opened the pull request:
CLDR-18410

- [ ] This PR completes the ticket.

<!--
Thank you for your pull request.
Please see https://cldr.unicode.org/index/process for general
information on contributing to CLDR.

1. Make sure the ticket is filed at
https://unicode-org.atlassian.net/projects/CLDR/
2. Update the PR title and first line of this
message to include the ticket ID (CLDR-_____)
3. You will be automatically asked to sign the contributors’
license before the PR is accepted.
- sign: https://cla-assistant.io/unicode-org/cldr
- license: https://www.unicode.org/copyright.html#License
-->

ALLOW_MANY_COMMITS=true

macchiati commented:
Just stubs for now (understood).

I think we want DDL Help pages under https://5adf953d.cldr.pages.dev/translation

srl295 commented:
> Just stubs for now (understood).
> 
> I think we want DDL Help pages under https://5adf953d.cldr.pages.dev/translation

how would you propose to organize these? I thought it was to be a parallel to that directory, as that is focussed on TC orgs?

macchiati commented:
What I don't want is for it to be buried down underneath a workgroup.

So it should be either visible in the contents panel on Information Hub for
Linguists <https://5adf953d.cldr.pages.dev/translation>  or in the contents
panel of the main page <https://5adf953d.cldr.pages.dev/>.

On Mon, Mar 17, 2025 at 1:51 PM Steven R. Loomis ***@***.***>
wrote:

> Just stubs for now (understood).
>
> I think we want DDL Help pages under
> https://5adf953d.cldr.pages.dev/translation
>
> how would you propose to organize these? I thought it was to be a parallel
> to that directory, as that is focussed on TC orgs?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/unicode-org/cldr/pull/4499#issuecomment-2730895153>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACJLEMEJK4POUCHGDHL2EOT2U4YVNAVCNFSM6AAAAABY67EO4KVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZQHA4TKMJVGM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
> [image: srl295]*srl295* left a comment (unicode-org/cldr#4499)
> <https://github.com/unicode-org/cldr/pull/4499#issuecomment-2730895153>
>
> Just stubs for now (understood).
>
> I think we want DDL Help pages under
> https://5adf953d.cldr.pages.dev/translation
>
> how would you propose to organize these? I thought it was to be a parallel
> to that directory, as that is focussed on TC orgs?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/unicode-org/cldr/pull/4499#issuecomment-2730895153>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACJLEMEJK4POUCHGDHL2EOT2U4YVNAVCNFSM6AAAAABY67EO4KVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZQHA4TKMJVGM>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

AEApple commented:
I thought we had discussed that the Info Hub would still be the main landing page, and then we would have a section or similar where it would say:
- 'contributing via an organization?' -> do this
- 'contributing as a guest?' -> do the other thing

Ideally even when people are contributing via organizations it is likely to our benefit to funnel information through the main organization contacts and have them funnel feedback back from their organizations contributors.

srl295 commented:
the original ask as i remember was "make a ddl landing page for ddl contributors so they know what to do"… maybe we can discuss tomorrow

srl295 commented:
TC Infra discussion: move ddl help pages under `/translation/ddl`

srl295 commented:
Mark had approved the structure. appreciate review on the content so we can get this in.

AEApple commented:
@srl295 - So are we holding off publishing this until after Monday's meeting since we're still discussing the organization form, or would you prefer to publish but without the organization form yet?

macchiati commented:
We'll be settling on the org form Monday, so we should publish without
waiting. It can go out with the ddl announcement

On Wed, Apr 23, 2025, 21:09 Annemarie Apple ***@***.***>
wrote:

> *AEApple* left a comment (unicode-org/cldr#4499)
> <https://github.com/unicode-org/cldr/pull/4499#issuecomment-2826317932>
>
> @srl295 <https://github.com/srl295> - So are we holding off publishing
> this until after Monday's meeting since we're still discussing the
> organization form, or would you prefer to publish but without the
> organization form yet?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/unicode-org/cldr/pull/4499#issuecomment-2826317932>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACJLEMGKSUA35SJW3MKZNFT23BPZDAVCNFSM6AAAAABY67EO4KVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDQMRWGMYTOOJTGI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>

srl295 commented:
So, ok to review and publish?

macchiati commented:
I was responding (I thought) about the blog post (not this PR). Just wanted to make that clear.

macchiati commented on docs/site/translation/ddl/organization.md:
```diff
@@ -0,0 +1,25 @@
+title: 'Requesting a new organization'
+---
+
+Please consider filling out our new form to request a CLDR Organization.
+
+> Note: This is a new form, so we’d appreciate feedback on it — you can add comments in the last question.
```
Rethinking this a bit; we need more text explaining what a CLDR Organization can do, what kinds of organizations we recognize, and what the obligations are.

Also, it would be good to have some cautions; having a CLDR Locale at Basic does not _guarantee_ that it will be picked up by products; for example, some may filter out languages without sufficiently large populations in countries they target.

srl295 replied:
Good feedback, thanks

macchiati replied:
I put together some draft text in https://github.com/unicode-org/cldr/pull/4644/files?short_path=2a327e4#diff-2a327e491ec5c16e9652f29369eef26b0a75840420239c4e7e08a737f6f029cf

srl295 commented:
I haven't addressed all of the feedback yet, however, I have:

-  merged in @macchiati changes 
- updated the org page to mention the form
- flattened all DDL pages to `/translation/ddl` (to be fleshed out)

macchiati commented on docs/site/translation/ddl.md:
```diff
@@ -0,0 +1,42 @@
+
+... please consider creating a [CLDR Organization] 
+
+### Users
+
+... then you can see [Managing users]...
+
+### If the locale is missing
+
+If you need to add a new locale that isn’t currently assigned to your organization, follow these instructions:
+
+1. Verify the locale is visible in the Survey Tool. If not, fill out the form: [Core Data for New Locales].
+
+2. [File a ticket] requesting to add the locale to your organization's `Locales.txt` file.
+
+3. When the locale is added to the Survey Tool and the Locales.txt file has been added, you can then grant your vetters access to that language. See [Managing users].
```
```suggestion
### Individual Contributors

1. Verify the locale is visible in the Survey Tool. 
    1. If not, fill out the form: [Core Data for New Locales].
    2. [QUESTION FOR STEVEN: How do people track progress? Eg, find out when their locale has been added? Whether there are problems with what they filled out?
2. Request a Survey Tool account with the form [XXXXXXXXX]

```



The rest really belongs in the main Managing Users document.
